package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "client",
-  "version": "2.6.0-v2.5",
+  "version": "2.6.0-v2.9",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "client",
-      "version": "2.6.0-v2.5",
+      "version": "2.6.0-v2.9",
```

Comment:
@sam-rice you don't need to change this, why do you update this manually? it should be down automatically, can you remove this change?

Reply:
Weird. I never manually changed this—not sure how it happened, but I'll change it back.